src/hooks/voucherHooks.ts

```diff
@@ -46,3 +49,60 @@ export const useVoucher = (): {
     memberId
   };
 };
+
+export const useRedeemLocalVouchers = () => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  const [redeemQuotaOfVoucher] = useMutation(REDEEM_QUOTA_OF_VOUCHER);
+
+  const redeemLocalVouchers = useCallback(async () => {
+    setIsLoading(true);
+    setIsError(false);
+
+    try {
+      const voucherTransactions = (await readFromStore(VOUCHER_TRANSACTIONS)) || [];
+      let error = false;
+
+      if (voucherTransactions.length) {
+        for (const { voucherId, memberId, quantity, deviceToken } of voucherTransactions) {
+          try {
+            await redeemQuotaOfVoucher({
+              variables: {
+                deviceToken,
+                quantity,
+                voucherId,
+                memberId
+              }
+            });
+          } catch (e) {
+            console.error(e);
+            error = true;
+            break;
+          }
+        }
+        if (!error) {
```

Comment:
please add a space before the condition starts

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/f2a3dfc7af2f00510b8f4a9ccd27aeb503edd0de